### DISCUSSION TITLE
How to create a project without signing in?

### DISCUSSION THREAD

**Discussion by aventus13** [Kong/insomnia#7335] [Help]
Hello! I exported all my collections to a json file, installed a new Insomnia instance and am trying to import the collections just as I was always able to. However, when trying to do it via Application > Preferences > Data I'm getting a message "There is no active project. Create a new project to import or export data". With the local scratchpad open, I can't figure out how to create a project. Can someone please help?*

\*I hope that the answer isn't _"You now have to use a signed-in account to create projects"_. I moved from Postman to Insomnia few years ago as I was looking for a hassle-free tool that isn't forcing its used to unnecessarily register.

**Comment by subnetmarco** [comment]
Scratchpad doesn't have "projects", it allows to create a local collection without having an account but it comes with limited capabilities. 

To create projects and get access to the other capabilities, you need to create a free account and login. After you do that, you can choose what projects can be stored locally or in the cloud, based on how you intend to collaborate with your files (collections, tests, design specs, etc).

As we are building more and more cloud capabilities for Insomnia, we are requiring users to register in order to get access to those features. After you login, the session will be remembered across application restarts so that it will not ask you to login every time you open Insomnia.

**Reply by domigi** [reply]
What if your so beloved users don't want access to your "planed" cloud capabilities? Why not give them the option to use Insomnia without an account?..

**Reply by aventus13** [reply]
@domigi I think that at this point it's not even worth fighting this battle. Clearly the Insomnia team made their call. If it works for them then so be it. However, it doesn't work for some users. The best thing we can do is switch to another tool. There are plenty tools available out there, some even open source. My team has already switched from Insomnia shortly after I posted this question. We're not going to waste time hoping that Insomnia might change its direction. It's just sad that Insomnia seems to be going Postman way- over bloating with features and forcing accounts sign in.
